src/style.css

```diff
@@ -2,2 +1,71 @@
+* {
   margin: 0;
+  padding: 0;
+}
+
+.html {
+  --header-link-color: #00acdc;
+}
```

Comment:
It's not common practice to create a class named `.html`. If you want to define CSS variables or set styles for the entire document, you should place them within the `:root` selector.